features/borrowers_view_and_sign_deed.feature

```diff
@@ -18,255 +18,67 @@ Scenario: Single borrower views and signs deed, then views
-Scenario: Single borrower views deed 3 times
-  Given I create default deed with <1> borrowers
-  And borrower <1> views the deed
-  And borrower <1> views the deed
-  Then borrower <1> views the deed
-
-@us133 @us133f
-Scenario: 2 Borrowers can view and sign a deed. p1
-  Given I create default deed with <2> borrowers
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
-  And borrower <2> views the deed
-  Then I verify borrower <1> has signed the deed
-  Then the deed is digitally signed by borrower <2>
-  And borrower <2> views the deed
-  Then I verify borrower <2> has signed the deed
-
-@us133 @us133f
-Scenario: 2 Borrowers can view and sign a deed. p2
-  Given I create default deed with <2> borrowers
-  And borrower <1> views the deed
-  And borrower <2> views the deed
-  Then the deed is digitally signed by borrower <2>
-  And borrower <1> views the deed
-  Then I verify borrower <2> has signed the deed
-  Then the deed is digitally signed by borrower <1>
-  And borrower <2> views the deed
-  Then I verify borrower <1> has signed the deed
-
-@us133 @us133f
-Scenario: 2 Borrowers can view and sign a deed. p3
-  Given I create default deed with <2> borrowers
-  And borrower <2> views the deed
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
-  And borrower <2> views the deed
-  Then I verify borrower <1> has signed the deed
-  Then the deed is digitally signed by borrower <2>
-  And borrower <1> views the deed
-  Then I verify borrower <2> has signed the deed
-
-@us133 @us133f
-Scenario: 2 Borrowers can view and sign a deed. p4
-  Given I create default deed with <2> borrowers
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
-  And borrower <2> views the deed
-  Then I verify borrower <1> has signed the deed
-  And borrower <2> views the deed
-  Then the deed is digitally signed by borrower <2>
-  And borrower <1> views the deed
-  Then I verify borrower <2> has signed the deed
-  Then I verify borrower <1> has signed the deed
-
-@us133 @us133f
-Scenario: 2 Borrowers can view and sign a deed. p5
+Scenario: 2 Borrowers can view and sign a deed.
   Given I create default deed with <2> borrowers
-  And borrower <2> views the deed
-  Then the deed is digitally signed by borrower <2>
   And borrower <1> views the deed
-  Then I verify borrower <2> has signed the deed
-  And borrower <1> views the deed
-  Then I verify borrower <2> has signed the deed
+  And the mortgage deed is displayed
   Then the deed is digitally signed by borrower <1>
   And borrower <2> views the deed
+  And the mortgage deed is displayed
   Then I verify borrower <1> has signed the deed
-  Then I verify borrower <2> has signed the deed
-
-@us133 @us133f
-Scenario: 3 Borrowers can view a deed. p1
-  Given I create default deed with <3> borrowers
-  And borrower <2> views the deed
-  And borrower <1> views the deed
-  Then borrower <3> views the deed
```

Comment:
Same comment as above regarding does this need to be removed as no signing is taking place?